src/cgi/mfs.cgi.in

```diff
@@ -3219,6 +3219,7 @@ if "HELP" in sectionset:
     # print_file("@CGI_PATH@/help.html")
     print("""please contact with help@lizardfs.com""")
     print("""<br/>""")
+    print """<iframe src="https://docs.lizardfs.com/", width="100%", height="100%"></iframe>"""
```

Comment:
```suggestion
    print("""<iframe src="https://docs.lizardfs.com/", width="100%", height="100%"></iframe>""")
```

Reply:
You're absolutely correct. My Spot on, my apologies in that regard.

Thanks for your patience, that was a novice mistake.